### ISSUE TITLE
Please add https://www.openhwgroup.org/core-v-devkits/ landing page

### ISSUE THREAD

**Issue by DBees** [openhwgroup/openhwgroup.org#463]
Please add the landing page for https://www.openhwgroup.org/core-v-devkits/.
This needs to be added this week and go live ideally on Sunday June 19

The content is https://docs.google.com/document/d/159bGcqvPy64pY1Fk2JRJHEdAEbUtW0bHWznOU33SHBk/edit#

There will be some minor updates coming to the content, see comments included.

**Comment by chrisguindon** [comment]
@DBees Sunday, June 19 is the weekend. I assume you would like us to make this live on or before Friday?

**Comment by DBees** [comment]
@chrisguindon If possible, if someone in Europe could make it go live Monday morning EU time that would be good. Will lineup with an announcement being made in Europe on Monday.

**Comment by ericpoirier** [comment]
@DBees should we add a link to this new page somewhere in the menu?

**Comment by ericpoirier** [comment]
@DBees Please review the following staging page:

https://deploy-preview-464--eclipsefdn-openhwgroup.netlify.app/core-v-devkits/

Again, note that there are currently no links to this page. Let me know if I should be adding a link in either the main menu or somewhere else on the site.

Thanks.

**Comment by DBees** [comment]
@ericpoirier the staging looks great. Yes we need to add "CORE-V DevKits" as a top level menu item. Thanks!

**Comment by DBees** [comment]
The DevKit image is updated @ericpoirier . Can you please substitute this one? https://mattermost.openhwgroup.org/all-users/pl/ffnghhsrkfyftrpayh5p5rpm4o. Let me know if you can't get the image from mattermost.

**Comment by ericpoirier** [comment]
> The DevKit image is updated @ericpoirier . Can you please substitute this one? https://mattermost.openhwgroup.org/all-users/pl/ffnghhsrkfyftrpayh5p5rpm4o. Let me know if you can't get the image from mattermost.

@DBees i'm not able to get the image.

**Comment by rickoco** [comment]
![OpenHW Demo Board No Background](https://user-images.githubusercontent.com/2498391/173918944-1e7d8750-5108-4de4-a9ab-c0774b20ee2d.png)
![OpenHW DevKit raytrace-4 final-with logo](https://user-images.githubusercontent.com/2498391/173918951-d398894a-0f7e-4047-b972-cf134cb6c573.png)
@ericpoirier here are updates to both board images

**Comment by rickoco** [comment]
Also @ericpoirier the title should have upper case V
From: CORE-v DevKits
To: CORE-V DevKits

**Comment by DBees** [comment]
@ericpoirier WRT Rick's images the first one replaces the CORE-V MCU Emulation Platform photo. The 2nd obviously is the updated CORE-V MCU DevKit image.

**Comment by ericpoirier** [comment]
@DBees I updated the images and added a link to the main menu.

https://deploy-preview-464--eclipsefdn-openhwgroup.netlify.app/core-v-devkits/

As for the capitalization in the title, I did a little bit of digging and we are currently using the following function on the title page and breadcrumbs in our main hugo theme.

https://gohugo.io/functions/title/

This is where we use it:
https://gitlab.eclipse.org/eclipsefdn/it/webdev/hugo-solstice-theme/-/blob/master/layouts/partials/main_prefix.html

I will submit a PR to our main hugo theme but this will be done separatly from this current PR.
